runtime/v2/runc/v2/service.go

```diff
@@ -689,15 +687,18 @@ func (s *service) getContainerPids(ctx context.Context, id string) ([]uint32, er
 	return pids, nil
 }
 
-func (s *service) forward(publisher events.Publisher) {
+func (s *service) forward(ctx context.Context, publisher shim.Publisher) {
+	ns, _ := namespaces.Namespace(ctx)
```

Comment:
Actually we don't need to create a new context here, wrapping `ctx` with `context.WithTimeout` returns a new context, which doesn't seem to affect the original `ctx` (based on my experiment).

However, I'm fine with doing the copy to be safe.

Reply:
I wanted to copy because I don't want to service context to cancel events publish

Reply:
It's the other way. I see.